content/index.md

```diff
@@ -74,4 +74,4 @@ with through the years.
 
 This site attempts to collect all the related facts, rationale and techniques for Trunk-Based Development together
 in one place, complete with twenty-five diagrams to help explain things. All without using TBD as an acronym
-even ~~once~~ twice.
+even ~~once~~ or twice.
```

Comment:
sure it does

I just feel it should still have an __or__ 

_Sent from my Huawei Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_